src/main/java/sx/blah/discord/handle/impl/obj/User.java

```diff
@@ -38,6 +38,17 @@
 public class User implements IUser {
 
 	/**
+	 * Default avatars for users
+	 */
+	private static String[] DEFAULT_AVATARS = {
```

Comment:
Still, how else would it be possible? As far as my awareness goes Discord payloads don't contain a default avatar field.